bundles/AdminBundle/Resources/views/SearchAdmin/Search/Quicksearch/object.html.twig

```diff
@@ -27,57 +27,49 @@
                     {% set break = true %}
                 {% endfor %}
             {% elseif definition is instanceof('\\Pimcore\\Model\\DataObject\\ClassDefinition\\Data\\Classificationstore') %}
-                {% set storedata = definition.getVersionPreview(element.getValueForFieldName(fieldName)) %}
+                {% set storedata = element.getValueForFieldName(fieldName) %}
 
-                {% set existingGroups = [] %}
                 {% set activeGroups = [] %}
 
                 {% if storedata %}
                     {% set activeGroups = storedata.getActiveGroups() %}
                 {% endif %}
-
-                {% for activeGroupId, enabled in activeGroups %}
-                    {% set arr = existingGroups|merge({activeGroupId: activeGroupId}) %}
-                {% endfor %}
-
-                {% if existingGroups is not empty %}
+                {% if activeGroups is not empty %}
                     {% set languages = ['default'] %}
                     {% if definition.isLocalized() %}
                         {% set languages = validLanguages|merge(languages) %}
                     {% endif %}
 
-                    {% for activeGroupId, enabled in existingGroups %}
-                        {% if activeGroups[activeGroupId] is not empty %}
-                            {% set groupDefinition = pimcore_object_classificationstore_group(activeGroupId) %}
-                            {% if groupDefinition is not empty %}
-                                {% set keyGroupRelations = groupDefinition.getRelations() %}
+                    {% for activeGroupId, enabled in activeGroups %}
+                        {% if activeGroups[activeGroupId] is empty %}
+                            continue;
+                        {% endif %}
```

Comment:
Are you sure the `continue;` works? Never seen that in Twig before.

Maybe a [filter](https://twig.symfony.com/doc/3.x/filters/filter.html) like this would work (not tested)?
```suggestion
                    {% for activeGroupId, enabled in activeGroups|filter(activeGroups[activeGroupId] is not empty) %}
```

Reply:
Good catch, seems to be a leftover from the php templates. I will update it, after checking if there are any other occurences. Thanks for the review/suggestion!